activities/domain/v1/domain.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+package io.spoud.sdm.activities.domain.v1;
+
+option java_multiple_files = true;
+option java_package = "io.spoud.sdm.activities.domain.v1";
+option go_package = "activities.domain.v1";
+
+import "google/protobuf/timestamp.proto";
+import "global/domain/v1/domain.proto";
+import "global/selection/v1/reference.proto";
+
+
+message Activity {
+  string id = 1;
+
+  // activity type
+  ActivityType.Type activity_type = 2;
+
+  // who causes the activity
+  io.spoud.sdm.global.selection.v1.EntityRef triggered_by = 4;
+
+  // who was affect by the change
+  io.spoud.sdm.global.selection.v1.EntityRef affected_entity = 5;
+
+  // pre build msg
+  string message = 6;
+
+  // old value
+  string old_value = 7;
+
+  // new value
+  string new_value = 8;
+
+  // extra entity ref
+  map<string, io.spoud.sdm.global.selection.v1.EntityRef> references = 9;
```

Comment:
what's the actual purpose of this ?

made me think of 
![image](https://user-images.githubusercontent.com/3659220/111644935-95d3df00-8800-11eb-962c-e5c5944f2f77.png)

Reply:
is to be use to resolving some entities outside like in Alfred. eg. if owner changes from A to B we need to know this two entities in order to resolve later but can be anything from any entity. any idea how to solve it differently?

Reply:
Sounds like a big mess to me to be honnest. Do we need this feature? knowing A and B for now? Can't we just start simple and not have too much in the API. I don't want to maintain stuff because we thought it would be a good idea when we started it